### PR TITLE
phpbu: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/phpbu.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Run backups using a specific configuration file:
+
+`phpbu --configuration={{path/to/configuration_file.xml}}`
+
+- Only run the specified backups:
+
+`phpbu --limit={{backup_task_name}}`
+
+- Don't actually create the backups:
```
Comment (schneiderl) [inline pages/common/phpbu.md]: Maybe rephrase this? 
This description does not say what the command does. If fact, it says exactly the opposite.

Comment (Aracki) [reply pages/common/phpbu.md]: but the command itself says `simulate`. I think this description can be left as it is :/

Comment (schneiderl) [reply pages/common/phpbu.md]: In my opinion you should not need to look at the example to understand the description. 
But if you guys feel that this is the best way i'll go with that.

Comment (andrik) [reply pages/common/phpbu.md]: I agree with @schneiderl

Comment (Aracki) [reply pages/common/phpbu.md]: Or "Simulate actions phpbu would have permormed"... I am ok with both examples...

**Comment by pxgamer** [comment]
I'm not sure it's really for backing up the code (although you could), more for backing up anything. It supports a wide range of databases, etc.

**Comment by sbrl** [comment]
Ah, I see! That makes much more sense :D
